docs/guides/end-to-end-testing/testing-strategies/social-authentication.mdx

```diff
@@ -0,0 +1,423 @@
+---
+title: Social Authentication
+slug: /guides/end-to-end-testing/social-authentication
+---
+
+:::info
+
+## <Icon name="graduation-cap"></Icon> What you'll learn
+
+- Log into your application via Google, Facebook, or Microsoft through the UI
+  with [`cy.origin()`](/api/commands/origin) using Auth0 social connections. The
+  same patterns will work with Github, LinkedIn, and others.
+- Understanding
+  [challenges](http://localhost:3000/guides/references/best-practices#When-logging-in)
+  with social connections
+
+:::
+
+:::caution
+
+<strong class="alert-header">
+  <Icon name="exclamation-triangle"></Icon> Not recommended in CI
+</strong>
+
+Cypress does **not** recommend testing social connection authentication as a
+primary means of application authentication testing. This is due to the
+[challenges](/guides/references/best-practices#Challenges-with-Social-Connection-Authentication)
+mentioned in our [Best Practices Guide](/guides/references/best-practice).
+
+If testing user based authentication flows via social connections is a **must**
+for your application, we recommend testing this in
+[open mode](https://docs.cypress.io/guides/guides/command-line#cypress-open) (or
+[run mode](https://docs.cypress.io/guides/guides/command-line#cypress-run)
+locally and
+[headed](https://docs.cypress.io/guides/guides/command-line#cypress-run-headed))
+to avoid A/B tests or alternate login flows. Relying on social authentication in
+CI will likely result in bot detection and in some cases, account suspension due
+to violating the provider's Terms of Service.
+
+:::
+
+## Application Setup
+
+For this guide, we are going to set up an application that uses Auth0 for
+authentication. Our Auth0 Tenant will have social connections set up for Google,
+Facebook, and Microsoft which with authenticate users on our behalf to our
+application.
+
+### Setting up the Tenant
```

Comment:
```suggestion
### Setting up the Auth0 tenant
```

Reply:
addressed in https://github.com/cypress-io/cypress-documentation/commit/ac6af55da2402a2a483d092323c97b44e6086712